.github/workflows/ci.yml

```diff
@@ -86,8 +87,15 @@ jobs:
       - uses: sfackler/actions/rustup@master
       - run: echo "::set-output name=version::$(rustc --version)"
         id: rust-version
-      - run: echo "VCPKG_ROOT=$env:VCPKG_INSTALLATION_ROOT" | Out-File -FilePath $env:GITHUB_ENV -Append
-      - run: vcpkg install openssl:x64-windows
+      - name: Prepare vcpkg
+        uses: lukka/run-vcpkg@v7
+        id: runvcpkg
+        with:
+          vcpkgArguments: openssl:x64-windows
+          vcpkgDirectory: ${{ runner.workspace }}/vcpkg/
```

Comment:
`runner.workspace` isn't documented as a key in the Actions docs.

I think this option only applies when using vcpkg as a submodule of the repository though, which we aren't doing.

Reply:
The `best practices` part documents

```
To sum up, you need to pin the specific version of vcpkg you want to use to keep a consistent development experience between local and remote build environments. This is accomplished by using vcpkg as submodule of your Git repository; this way the version of vcpkg used is implied by the commit id specified by the submodule for vcpkg.
```

If you remove it, it will download the  vcpkg git repository each time, which makes no sense

https://github.com/marketplace/actions/run-vcpkg#best-practices

Reply:
But the directory you are pointing to here is not a submodule containing a vcpkg repository.

Reply:
Yes, I think adding the whole vcpkg repo as git submodule bloats more and not git clone friendly, that's why the github actions requires to pin at a specific commit.